### ISSUE TITLE
Staging Review finding: "Where to send the payment" question may be unclear to screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99260]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-7959f-2, Foreign Medical Program claim cover sheet
**Feature:** 

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/file-claim-form-10-7959f-2/payment-selection](https://staging.va.gov/health-care/foreign-medical-program/file-claim-form-10-7959f-2/payment-selection), the label for the question prompt is "Tell us where to send the payment for this claim (*Required)," then there's some helper text (two bullet points describing options), then a rephrasing of the question prompt "Send payment to:" and then finally the radio button options.

Visually, this is pretty easy to follow. But for a screen reader user tabbing into the first radio option, the only text that will be announced will be "Tell us where to send the payment for this claim (*Required)" followed by the radio options --- with all of the helper text in the middle skipped, including the explanation about payments being a reimbursement to you or a direct payment to the provider.

<details><summary>Link, screenshot or steps to recreate</summary>

![Step 6 of 8, Where to send payment, with question prompt "Tell us where to send the payment for this claim" followed by two bullet points of explanation, a re-prompt "Send payment to" and then radio options for Veteran and Provider.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/77082-Brian-1.png?raw=true)

</details>

## Recommended action

There are two easy solutions here.

The option that I think is easier and quicker: Use the [`tile` variant for `va-radio-option`](https://design.va.gov/storybook/?path=/docs/uswds-va-radio--docs#tile), which supports a `description` prop that adds helper text in-context with the radio option itself. The grammar of the content may need to change, but you can include the information about when to choose each option with each option. `description` is structured to associate that text with the relevant radio option using `aria-describedby` for screen reader users.

The other option that also works: Move the helper text into the `legend` of the `fieldset` for this page, grouped in with "Where to send the payment." You can style the "Where to send the payment" heading and the helper text however you'd like, and moving the helper text to the `legend` satisfies the requirement to programmatically associate the text with the radio buttons.

Either are fine! But as currently coded there's some important context that's missing.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
